Zulip /README.md

```diff
@@ -0,0 +1,42 @@
+# Hi :smiley: :wave:
+
+
+## If you’re new here, here’s what you can do to get started with this community:
+
+:heavy_check_mark: [Introduce yourself on #newcomers in the #newcomers > Introductions topic and tell us your motivations for joining so we can get to know you and help you on your journey](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers)
+
+:calendar: [Follow our Open Sessions in Calendar Events topic on #announcements or through our AnitaB.org Open Source calendar (PST:clock1:). These are voice calls you can participate in and get to know a bit more our community and projects](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles)
+
+:eyeglasses: Learn about our [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/code_of_conduct.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/reporting_guidelines.md) to keep this community safe for you and others.
+
+# [To get Acquainted with Zulip](https://zulipchat.com/help/getting-started-with-zulip)  
```

Comment:
you could put this under a subsection called "Learning Resources", not a bullet point